File: 2026/01/06/main.java
import java.io.*;
import java.util.ArrayList;
import java.util.List;

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        List<String> lines = new ArrayList<>();
        for(int i=0; i<3;i++){
            lines.add(br.readLine());
        }

        for(String line: lines){
            String prev = line.substring(0, 1);
            String[] rest = line.substring(1).split("");
            int count = 1;
            int answer = 1;

            for(String s: rest){
                if(prev.equals(s)){
                    count += 1;
                } else {
                    count = 1;
                    prev = s;
                }

                answer = Math.max(answer, count);
            }
            System.out.println(answer);

        }

    }
}

File: 2026/01/06/main.py
lines = [input() for _ in range(3)]

for nums in lines:
    count = 1
    answer = 1
    prev = nums[0]
    for n in nums[1:]:

        if prev == n:
            count += 1
        else:
            count = 1
            prev = n

        answer = max(count,answer)

    print(answer)



